docs/providers/azure-ad.md

```diff
@@ -14,42 +14,42 @@ https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-regis
 ## Example
 
 ### To allow specific Active Directory users access:
-- In https://portal.azure.com/ -> Azure Active Directory create a new App Registration.
+- In https://portal.azure.com/ search for "Azure Active Directory", and select your organization.
+- Next, go to "App Registration" in the left menu, and create a new one.
 - Pay close attention to "Who can use this application or access this API?"
-  - This allows you to scope access to specific types of user accounts.
-- Make sure to remember / copy
+  - This allows you to scope access to specific types of user accounts
+  - Only your tenant, all azure tenants, or all azure tenants and public Microsoft accounts (Skype, Xbox, Outlook.com, etc.)
+- When asked for a redirection URL, use `https://yourapplication.com/api/auth/callback/azure-ad` or for development `http://localhost:3000/api/auth/callback/azure-ad`.
+- After your App Registration is created, under "Client Credential" create your Client secret.
+- Now copy your:
   - Application (client) ID
   - Directory (tenant) ID
-- When asked for a redirection URL, use http://localhost:3000/api/auth/callback/azure-ad
-- Create a new secret and remember / copy its value immediately, it will disappear.
+  - Client secret (value)
 
-In `.env.local` create the follwing entries:
+In `.env.local` create the following entries:
 
 ```
-AZURE_AD_CLIENT_ID=<copy Application (client) ID here> 
-AZURE_AD_CLIENT_SECRET=<copy generated secret value here>
+AZURE_AD_CLIENT_ID=<copy Application (client) ID here>
+AZURE_AD_CLIENT_SECRET=<copy generated cleint secret value here>
 AZURE_AD_TENANT_ID=<copy the tenant id here>
 ```
-**Note**: Omit the `AZURE_TENANT_ID` if you created the App Registration for:
-> Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)
-
 That will default the tenant to use the `common` authorization endpoint. [For more details see here](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-protocols#endpoints).
 
+:::note
+Azure AD returns the profile picture in an ArrayBuffer, instead of just a URL to the image, so our provider converts it to a base64 encoded image string and returns that instead.
```

Comment:
Let's add a link to https://docs.microsoft.com/en-us/graph/api/profilephoto-get?view=graph-rest-1.0#examples

Reply:
Sounds good. Added!